tests/robot/OBSERVATION/10_create_sofa_score.robot

```diff
@@ -302,7 +302,7 @@ ${randinteger}                  ${12345}
 	...                 6. *VALIDATE* the response status \n\n
     ...                 7. *VALIDATE* outcome against diagnostic text & location
 	[Template]			create sofa score with ehr reference
-    [Tags]              valueCodeableConcept
+    [Tags]              valueCodeableConcept    009
```

Comment:
Why did you add this tag?

Reply:
@af-pwo to be able to run just that specific test by tag